_studio/shared/umc/codec/h264_dec/include/umc_h264_bitstream_headers.h

```diff
@@ -228,6 +230,8 @@ class H264HeadersBitstream : public H264BaseBitstream
 
     Status GetPredWeightTable(UMC_H264_DECODER::H264SliceHeader *hdr, const UMC_H264_DECODER::H264SeqParamSet *sps,
         UMC_H264_DECODER::PredWeightTable *pPredWeight_L0, UMC_H264_DECODER::PredWeightTable *pPredWeight_L1);
+
+    inline bool CheckLevel(uint8_t profile_idc, uint8_t level_idc);
```

Comment:
no sense of ```inline``` key word. All class member functions are inline by default

Reply:
`CheckLevel` doesn't use any non-public state of `H264HeadersBitstream`, it 's not necessary to make it a class method, I'd rather make it free function.